sync:
  image: n3llyb0y/wait
  links:
    - auth
    - call-logd
    - postgres
  environment:
    PORTS: "9298 9497 5432"
    TIMEOUT: "${INTEGRATION_TEST_TIMEOUT}"

auth:
  image: p0bailey/docker-flask
  ports:
    - "9497"
  volumes:
    - "./xivo-auth:/usr/local/share/xivo-auth"
    - "./ssl:/usr/local/share/ssl"
  command: "python /usr/local/share/xivo-auth/mock-xivo-auth.py 9497"

call-logd:
  image: xivo-call-logd-test
  volumes:
    - "../../..:/usr/src/xivo-call-logs"
    - "./etc/xivo-call-logd:/etc/xivo-call-logd"
    - "./ssl:/usr/local/share/ssl"
  ports:
    - "9298"
  links:
    - auth
    - postgres

postgres:
  image: wazopbx/postgres-test
  ports:
    - "5432"
